core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -585,6 +585,21 @@ package object config {
       .intConf
       .createWithDefault(128)
 
+  private[spark] val LISTENER_BUS_LOG_SLOW_EVENT_ENABLED =
+    ConfigBuilder("spark.scheduler.listenerbus.logSlowEvent.enabled")
+      .doc("When enabled, log the event that takes too much time to process. This helps us " +
+        "discover the event types that cause performance bottlenecks. The time threshold is " +
+        "controlled by spark.scheduler.listenerbus.logSlowEvent.threshold.")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
`false` by default.

Reply:
Since this is really useful in tracking event lost related issues, and it's light-weighted, I'd recommend turn it on by default.

Reply:
+1 on true by default, since the issue might not be reproducible easily and it is only adding the overhead of number comparison in happy case (overhead of bad case shouldn't matter since if it occurs frequently it's likely having real issue soon). Moreover, if I understand correctly, we can't change it from outside of Spark while application is running - that means application should be stopped and restarted due to turn on this config.